src/components/logs/app/main.js

```diff
@@ -124,10 +121,11 @@ function debounce(func, wait, immediate) {
     };
 }
 
-function createElement(type, value, content) {
+function createElement(type, value, label) {
     const element = document.createElement(type);
-    if (value) { element.value = value; }
-    if (content) { element.textContent = content; }
+    if (value) { element.setAttribute('value', value); }
+    if (label) { element.textContent = label; }
```

Comment:
basically the problem here was that the 'value' attribute doesn't get set on the element. So when `vscode-single-select` goes to find the option that's selected, it can only use the content value